internal/pkg/vm/internal/storagemap/storagemap_test.go

```diff
@@ -1,337 +0,0 @@
-package storagemap
-
-import (
-	"testing"
-
-	"github.com/ipfs/go-datastore"
-	blockstore "github.com/ipfs/go-ipfs-blockstore"
-	cbor "github.com/ipfs/go-ipld-cbor"
-
-	"github.com/filecoin-project/go-filecoin/internal/pkg/encoding"
-	tf "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/testflags"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/actor"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/address"
-	internal "github.com/filecoin-project/go-filecoin/internal/pkg/vm/internal/errors"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestGetAndPutWithEmptyStorage(t *testing.T) {
```

Comment:
It would be nice to adapt some of these high level test conditions to the new untested storage.go stuff as there is a decent amount of conceptual overlap.